Packages/vcs/Lib/configurator.py

```diff
@@ -381,35 +365,26 @@ def place(self):
         if self.target:
             self.target.place()
 
-    def activate(self, obj, display):
-        if self.target is not None and self.shift() == False:
+    def activate(self, display, actor, key):
+        if self.target is not None and self.shift() is False:
             self.deactivate(self.target)
 
         self.toolbar.hide()
 
-        if display.g_type == "fillarea":
```

Comment:
I removed the unused editor types from this function; most of them should find their way back in 2.3, when I can finish creating quality implementations of everything. Since I changed the logic of how these objects are getting initialized, I decided to just dump the useless code rather than hang onto it anymore.